src/plugins/flexibleLayout/components/flexible-layout.scss

```diff
@@ -30,23 +29,26 @@
     flex: 0 0 auto;
   }
 
+  &--cols {
+  }
+
+  &--rows {
+  }
```

Comment:
We should remove these empty rulesets if they aren't being used

Reply:
Good catch, removed.